Sources/TensorFlow/Operators/Math.swift

```diff
@@ -424,6 +424,20 @@ internal func _vjpLog<T: TensorFlowFloatingPoint>(
     return (log(x), { v in v / x })
 }
 
+/// Computes `log(1 - exp(x))` using a numerically stable approach.
+///
+/// The approach is shown in Equation 7 of:
+/// https://cran.r-project.org/web/packages/Rmpfr/vignettes/log1mexp-note.pdf.
+@inlinable
+@differentiable
+public func log1mexp<T: TensorFlowFloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    let isTooSmall = (-x .< T(log(2.0))).withoutDerivative()
```

Comment:
```suggestion
    let isTooSmall = (-x.withoutDerivative() .< T(log(2.0)))
```

Reply:
This fix still does not work actually because the resulting tensor is still boolean.